### ISSUE TITLE
Discrepency between Anki Options and Anki manual: Minimum Interval

### ISSUE THREAD

**Issue by joshdavham** [ankitects/anki-manual#294]
There's currently a descrepency between how the Anki Options pane describes the Minimum Interval parameter and how the Anki manual describes it:

Anki Options pane (version 24.06.3):
<img width="1510" alt="min_int_options" src="https://github.com/user-attachments/assets/633d0181-4fde-4be6-b93a-f708314d30c9">

Anki manual:
<img width="1512" alt="min_int_manual" src="https://github.com/user-attachments/assets/6a9304b2-4bdf-4033-bddf-51396eaf7b3b">

I'm still learning how the default Anki scheduler works so I'm actually not sure which one is correct.

If anyone knows which is the correct one, I'd be happy to open a PR to correct it.

**Comment by joshdavham** [comment]
How I interpret the tooltip is that, if I have a review-state card and I hit again, it will be due no earlier than the minimum interval.

The manual on the other hand gives the interpretation that, once a relearning-state card has been rated good(and maybe also easy?) on its last relearning step, it will be due no later than the minimum interval.

These two interpretations are both plausible and incompatible.

**Comment by brishtibheja** [comment]
> it will be due no later than the minimum interval.

It's the default behaviour as by default `new interval` = `0.00`. If you read that topic later, maybe it becomes clear (well, I have really ignored those sm-2 parts of docs).

**Comment by brishtibheja** [comment]
The manual is correct. The default `new interval` resets the delay to `0`. But with the default `minimum interval` it jumps back to 1 day.

**Comment by joshdavham** [comment]
> The manual is correct.

Ok.

But is the manual also correct in the section describing the Minimum Interval? As I've demonstrated, the descriptions of the Minimum Interval between the manual and the Anki tooltip are discrepant.

**Comment by brishtibheja** [comment]
> But is the manual also correct in the section describing the Minimum Interval?

It is but it's slightly confusing because it implied setting `minimum interval` to `1d` makes your interval after relearning 1 day even though that may increase with `new interval`. You can fix that by making the end part, "meaning once relearning is finished the card has at least a `1d` interval" or something better.

(though, imo we're better off spending time rewriting other parts of the docs)

**Comment by joshdavham** [comment]
> once relearning is finished the card has at least a 1d interval

Is this correct? 

If so, the minimum interval does a lot more than what is communicated in the tooltip or the manual alone.

**Comment by brishtibheja** [comment]
> If so, the minimum interval does a lot more than what is communicated in the tooltip or the manual alone.

I'm not sure. I think that's what the manual/tooltip is communicating. If you have something in mind, you can do a PR.

**Comment by joshdavham** [comment]
>  If you have something in mind, you can do a PR.

I'm not going to open a PR until I understand what the minimum interval actually does. It's why I opened this issue.

**Comment by Danika-Dakika** [comment]
@joshdavham You're right that the "minimum interval" descriptions seem inconsistent. (The "new interval" doesn't really enter into it.)

I think the only thing that's missing though is that the tooltip doesn't mention the Relearn state -- which is technically optional (see https://docs.ankiweb.net/deck-options.html#relearning-steps).
